src/main/scala/no/ndla/articleapi/service/converters/contentbrowser/VideoConverterModule.scala

```diff
@@ -24,11 +24,11 @@ trait VideoConverterModule {
     override val typeName: String = "video"
 
     override def convert(content: ContentBrowser, importStatus: ImportStatus): Try[(String, Seq[RequiredLibrary], ImportStatus)] = {
-      val (linkText, nodeId) = (content.get("link_text"), content.get("nid"))
+      val (linkText, nodeId, linkContext) = (content.get("link_text"), content.get("nid"), content.get("open_in"))
```

Comment:
Samme kommentar som i ´GeneralContentConverterModule`. Tror ikke det finnes noe contentbrowserfelt "open-in".